test/androidTest/java/org/thoughtcrime/securesms/espresso/ConversationHelper.java

```diff
@@ -89,6 +93,36 @@ public ConversationHelper sendReaction(int position, int emoji) {
         return this;
     }
 
+    public ConversationHelper sendReply(int position, String message) {
+
+        this.unselectMessage();
+        this.selectMessage(position);
+
+        onView(withId(R.id.menu_context_reply))
+                .perform(click());
+        onView(withId(R.id.lin_reply))
+                .perform(click());
+        onView(withId(R.id.custom_reply))
+                .perform(typeText(message))
+
```

Comment:
is this space intentional? it feels awkward

Reply:
yes its for readability